utils/learn-lm.json

```diff
@@ -623,36 +623,70 @@
                             "level": 3,
                             "position": 1,
                             "isDir": false,
-                            "url": "/learn/observe-ballerina-programs",
-                            "id": "observe-ballerina-programs"
+                            "url": "/learn/overview-of-ballerina-observability",
+                            "id": "overview-of-ballerina-observability"
                         },
                         {
-                            "dirName": "Observe logs",
+                            "dirName": "Supported observability tools and platforms",
                             "level": 3,
                             "position": 2,
-                            "isDir": false,
-                            "url": "/learn/observe-logs",
-                            "id": "observe-logs"
-                        },
-                        {
-                            "dirName": "Observe metrics",
-                            "level": 3,
-                            "position": 3,
-                            "isDir": false,
-                            "url": "/learn/observe-metrics",
-                            "id": "observe-metrics"
-                        },
-                        {
-                            "dirName": "Observe tracing",
-                            "level": 3,
-                            "position": 4,
-                            "isDir": false,
-                            "url": "/learn/observe-tracing",
-                            "id": "observe-tracing"
+                            "isDir": true,
+                            "url": "/learn/supported-observability-tools-and-platforms",
+                            "id": "supported-observability-tools-and-platforms",
+                            "subDirectories": [
+                                {
+                                    "dirName": "Prometheus",
+                                    "level": 4,
+                                    "position": 1,
+                                    "isDir": false,
+                                    "url": "/learn/supported-observability-tools-and-platforms/prometheus",
+                                    "id": "prometheus"
+                                },
+                                {
+                                    "dirName": "Jaeger",
+                                    "level": 4,
+                                    "position": 2,
+                                    "isDir": false,
+                                    "url": "/learn/supported-observability-tools-and-platforms/jaeger",
+                                    "id": "jaeger"
+                                },
+                                {
+                                    "dirName": "Zipkin",
+                                    "level": 4,
+                                    "position": 4,
```

Comment:
Position:4 is repeated from here onwards. Add the correct value.

Reply:
@sm1990 corrected.